pkg/services/rendering/rendering.go

```diff
@@ -67,6 +67,7 @@ func (rs *RenderingService) Run(ctx context.Context) error {
 	}
 
 	if plugins.Renderer == nil {
+		rs.log.Info("Backend rendering via phantomJS")
```

Comment:
Not related to this pr and think the existing log is enough: https://github.com/grafana/grafana/blob/master/pkg/services/rendering/phantomjs.go#L18

Reply:
The log in `phantomjs.go#L18` only prints once you try to render something, this one helps to know what kind of rendering service was registered. Both http server backend and plugin announces their registration so you could know that by absence of those logs but this makes it a bit easier when testing plugins as it can happen to you that the rendering works just because the plugin was not registered.

Reply:
Make sense. Great catch :+1: